packages/module/patternfly-docs/content/extensions/chatbot/examples/UI/UI.md

```diff
@@ -342,3 +346,11 @@ Based on the [PatternFly modal](/components/modal), this modal adapts to the Cha
 ```js file="./ChatbotModal.tsx" isFullscreen
 
 ```
+
+### Settings
+
+A settings layout can be substituted for other components within the chatbot. It should accept any number of buttons, dropdowns, toggles, etc. and labels, and render them appropriately within all four display modes.
```

Comment:
```suggestion
To contain user preference controls and other ChatBot setting options, you can create a separate settings page that can accept any number of buttons, dropdown menus, toggles, labels, and so on. This settings page will render all components appropriately within all 4 display modes.

In this demo, you can toggle the settings page by clicking the "Launch settings" button in the display mode menu.
```
how does this rephrasing sound? 

not sure if "settings page" is entirely accurate, but also not sure what else to call this? Panel or menu seem equally as vague in this context, but as long as the main point gets across I'm fine with any of them.

also, what other places could we expect devs to trigger this page, besides via the options menu? Wondering mostly for the sake of determining if we should expand on this in the design guidelines in a follow up 🤔

Reply:
I think right now it's just the options menu.